starport/services/chain/chain.go

```diff
@@ -439,3 +452,11 @@ func (c *Chain) Commands(ctx context.Context) (chaincmdrunner.Runner, error) {
 
 	return chaincmdrunner.New(ctx, cc, ccroptions...)
 }
+
+// AppBackendConfigWatchPaths returns the files to watch for config changes
+func (c *Chain) AppBackendConfigWatchPaths() []string {
```

Comment:
https://github.com/tendermint/starport/pull/761#pullrequestreview-591864052 can eliminate this func as well.